packages/web-react/src/types/dictionaries.ts

```diff
@@ -0,0 +1,2 @@
+export type DictionarySizeType = 'small' | 'medium' | 'large';
+export type DictionarySizeExtendedType = 'xsmall' | 'xlarge';
```

Comment:
Please look at `dropdown.ts` where the `DropdownPlacements` type is defined. We want to define the dictionary the same way.

```ts
export const SIZES = {
  SMALL: 'small'
  // ...
} as const

export type SizesKeys = keyof typeof SIZES;
// using generic for extension
export type Size<E> = typeof SIZES[SizesKeys] | E;

// Now we can use both `small` or Size.SMALL :-)
```

Reply:
ok, i will do it like this